### PR TITLE
remove the note about dropping the database

### PR DESCRIPTION
**Pull request by renetapopova**
I removed the note but left the step 4. Do you think I should add the same step to the section **Restore a database backup**?

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1454.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1454.surge.sh/operations-manual/5/backup-restore/restore-backup)

**Comment by au-arena** [comment]
>  Do you think I should add the same step to the section Restore a database backup?

No, I don't think so.

**Review by au-arena — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1454.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1454.surge.sh/operations-manual/5/backup-restore/restore-backup)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1454.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1454.surge.sh/operations-manual/5/backup-restore/restore-backup)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1454.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1454.surge.sh/operations-manual/5/backup-restore/restore-backup)

**Review by au-arena — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/backup-restore/restore-backup.adoc [hunk 1]
```diff
@@ -143,18 +143,24 @@ That means, if you restore `neo4j-2023-06-29T14-50-45.backup`, your database wil
...
 +
 The `--from-path=` argument must contain the path to the last backup of a chain, in this case, `neo4j-2023-06-29T14-51-33.backup`.
-If you want to restore several databases at once, you can specify a comma-separated list of paths to backup artifacts.
-
-. Unless you are replacing an existing database, create the new database using `CREATE DATABASE` against the `system` database.
++
+[TIP]
+====
+If you want to restore several databases at once, you must stop them first and then you can alter the command by specifying a comma-separated list of paths to backup artifacts, and remove the `<database>` parameter.
+You should also skip the `CREATE DATABASE` step afterward.
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/backup-restore/restore-backup.adoc:161]: ah, ok.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1454.surge.sh

Updated pages:
- [Restore a database backup](https://neo4j-docs-operations-1454.surge.sh/operations-manual/5/backup-restore/restore-backup)

**Pull request merged**